src/utils.ts

```diff
@@ -380,18 +380,31 @@ export function computeAppLocationFromArtifactLocation(appArtifactLocation: stri
   return undefined;
 }
 
-export function getBin(binary: string) {
-  if (binary.indexOf(path.sep) >= 0) {
-    return path.isAbsolute(binary) ? binary : path.resolve(binary);
-  }
-
-  const binDirOutput = spawnSync(isWindows() ? "npm.cmd" : "npm", ["bin"], { cwd: process.cwd() });
+function exec(cmd: string, args: string[] = []) {
+  const binDirOutput = spawnSync(cmd, args, { cwd: process.cwd() });
   const binDirErr = binDirOutput.stderr.toString();
   if (binDirErr) {
-    console.error({ binDirErr });
+    console.error(`ERRR: `, binDirErr);
+    process.exit(-1);
+  }
+  return binDirOutput.stdout.toString().trim();
+}
+
+export function getBinaryPath(binary: string) {
+  const binDirOut = exec(isWindows() ? "npm.cmd" : "npm", ["bin"]);
```

Comment:
You don't need to search for `npm.cmd` on Windows, assuming that `npm` is correctly in the path it will take care of it for you